### ISSUE TITLE
Feruz: DuckDuckGo - First part

### ISSUE THREAD

**Issue by Feruzteame** [Feruzteame/duck-duck-clone#5]
# HTML
- [x] heading
* logo (download image and put DuckDuckGo logo)
* header(DuckDuckGo)
- [x] Form
* search button(This part is imporatnt part of web-page.should have good accessibility by puting * placeholder and Arial-label aria-label="search box".
* submit button(submit icon)
- [x] description
* text(Text)
* Button(Button)
- [x] star rated icon(icon)

# CSS
- [x] form{display: flex} to put search button and submit button side by side.
* form[placeholder]{font-size: bigger} to be accessible.
* I will try to inspect the real web-page to find the right color as the real DuckDuckGo web-site.

# Add media-quire

#screen-shot

![Screenshot 2020-06-01 at 22 11 51](https://user-images.githubusercontent.com/59234162/83451077-b344e180-a456-11ea-9dd6-c9427b581c40.png)
